### PR TITLE
clean up API

### PR DESCRIPTION
**Pull request by jburel**
Further cleanup
Fix link to annotation that did not make sense
https://trello.com/c/tF1xM4XW/48-minor-api-doc-fix

### CODE REVIEW

File: omero/developers/Modules/Api.txt [hunk 1]
```diff
@@ -14,16 +14,16 @@ connection e.g. Python:
     adminService = session.getAdminService()                 # now we can get/create services
 
 -  The :javadoc:`Service factory API <slice2html/omero/api/ServiceFactory.html>`
-   has methods for creating Stateless and Stateful services (see below).
+   has methods for creating Stateless and Stateful services see below.
```
Comment (hflynn) [inline omero/developers/Modules/Api.txt]: The brackets were fine here, you'll need a comma after 'services' now you've removed it

**Comment by mtbc** [comment]
Staged at http://www.openmicroscopy.org/site/support/omero5.3-staging/developers/Modules/Api.html.

File: omero/developers/Modules/Api.txt [hunk 2]
```diff
@@ -105,21 +105,21 @@ Stateful/Binary Services
 
 -  RawFileStore:
    :source:`src <components/common/src/ome/api/RawFileStore.java>`,
-   :javadoc:`API <ome/api/RawFileStore.html>`
+   :javadoc:`API <ome/api/RawFileStore.html>` for reading files
```
Comment (mtbc) [inline omero/developers/Modules/Api.txt]: also writing?

File: omero/developers/Modules/Api.txt [hunk 3]
```diff
@@ -105,21 +105,21 @@ Stateful/Binary Services
...
 -  RawPixelsStore:
    :source:`src <components/common/src/ome/api/RawPixelsStore.java>`,
-   :javadoc:`API <ome/api/RawPixelsStore.html>`
+   :javadoc:`API <ome/api/RawPixelsStore.html>` for reading pixels data
```
Comment (mtbc) [inline omero/developers/Modules/Api.txt]: also writing?

File: omero/developers/Modules/Api.txt [hunk 4]
```diff
@@ -143,16 +143,16 @@ permissions, otherwise an exception is thrown.
 
 Dirty checks follow the Three Commandments:
 
-#. Any IObject-valued field with unloaded set to true is treated as a
-   place holder (proxy) and is re-loaded from the database.
+#. Any IObject-valued field with unloaded set to true is treated as proxy
+   and is re-loaded from the database.
```
Comment (mtbc) [inline omero/developers/Modules/Api.txt]: reloaded

File: omero/developers/Modules/Api.txt [hunk 5]
```diff
@@ -180,55 +180,55 @@ Examples
...
 obtained, a stateful service proxy can only be used by a single user. After
-task completion, the service should be closed (``proxy.close()``) to free up
+task completion, the service should be closed i.e. ``proxy.close()`` to free up
 server resources.
 
 How to write a service
 ----------------------
 
 A tutorial is available at :doc:`/developers/Server/HowToCreateAService`. In
 general, if a properly annotated service is placed in any JAR of the OMERO EAR
```
Comment (mtbc) [inline omero/developers/Modules/Api.txt]: there's an OMERO EAR file? :smiley:

File: omero/developers/Modules/Api.txt (hunk 4 above)
Comment (mtbc) [reply omero/developers/Modules/Api.txt]: "as *a* proxy"

**Review by mtbc — APPROVED** [review]
Thank you, I regard this as one of our more important pages.

**Pull request merged**